tests/test_residuals.py

```diff
@@ -438,3 +438,46 @@ def test_resid_mean_phase():
     mean, _ = pint.residuals.weighted_mean(full, w)
     assert mean == r.calc_phase_mean()
     assert r.calc_phase_mean() == r2.calc_phase_mean()
+
+
+@pytest.mark.parametrize(
```

Comment:
This is fine I guess, but is it necessary if you aren't actually iterating over multiple inputs?

Reply:
The idea here is to make sure it works without and with red noise.

Reply:
ok, I had mis-read it.  I now see there are 2 versions.